spartan/aztec-network/templates/pxe.yaml

```diff
@@ -123,7 +123,7 @@ spec:
                 - |
                   curl -s -X POST -H 'content-type: application/json' \
                   -d '{"jsonrpc":"2.0","method":"pxe_getNodeInfo","params":[],"id":67}' \
-                  127.0.0.1:{{ .Values.pxe.service.nodePort }} | grep -q '"protocolVersion":1'
+                  127.0.0.1:{{ .Values.pxe.service.nodePort }} | grep -q '"protocolVersion":[0-9]*[1-9][0-9]*'
```

Comment:
[1-9][0-9]* ?

Reply:
Ye problably better, should not be with a 0 start if number so don't have to cover 👍